src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final double MIN_SIZE = 1.0;
+    public static final double MAX_SIZE = 15.0;
+    public static final double DEFAULT_RADIUS = 10.0;
+
+    private Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+
+        switch (figureNumber) {
+            case 0: {
+                final double side = random.nextDouble(MIN_SIZE, MAX_SIZE);
+                return new Square(colorSupplier.getRandomColor(), side);

```

Comment:
same for other figures
```suggestion
                return new Square(randomColor, side);
```